code/modules/admin/tabs/admin_tab.dm

```diff
@@ -185,6 +185,28 @@
 	dat += "</body></html>"
 	show_browser(usr, dat, "Admin record for [key]", "adminplayerinfo", "size=480x480")
 
+/datum/admins/proc/check_ckey(target_key as text)
+	set name = "Check CKey"
+	set category = "Admin"
+
+	var/mob/user = usr
+	if (!istype(src, /datum/admins))
+		src = user.client.admin_holder
+	if (!istype(src, /datum/admins) || !(rights & R_MOD))
+		to_chat(user, "Error: you are not an admin!")
+		return
+	if(!target_key)
```

Comment:
```suggestion
	target_key = ckey(target_key)
	if(!target_key)
```
Since it looks like this check was case sensitive (e.g. searching for `Drathek` had no results), you may as well use the ckey proc to lowercase and remove symbols that aren't allowed. E.g. `Drathek.` would search `drathek`

Done here in case the result would then be nothing (e.g. `.` will say no key detected)